src/use-cases/createUseCase.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, describe, it } from "vitest";
+import { CreateUseCase } from "./createUseCase";
+import { compare } from "bcryptjs";
+import { InMemoryUserRepository } from "../repositories/in-memory-db/inMemoryUserRepository";
+import { UserAlreadyExistsError } from "./errors/user-already-exists-error";
+
+describe('Register Use Case', () => {
+  it('should be able to register', async () => {
+    const usersRepository = new InMemoryUserRepository
+    const createUseCase = new CreateUseCase(usersRepository)
+
+    const { user } = await createUseCase.execute({
+      name: 'John',
+      surname: 'Doe',
+      email: 'johndoe@email.com',
+      password: '123456',
+    })
+
+    expect(user.id).toEqual(expect.any(String))
```

Comment:
I think there is a better way to test that. Since the IDs are generated and you do not control them, you can compare the email or any other information that you know previously to create the user.